src/editor/screens/LessonPlanEditor.js

```diff
@@ -10,22 +10,25 @@ const LessonPlanEditor = ({ navigation }) => {
     <SafeAreaView style={{ backgroundColor: '#FFFAF5' }}>
       <LessonPlanHeader />
       <ScrollView style={styles.scrollView}>
-        <View style={{ flexDirection: 'column', alignItems: 'center' }}>
-          <LessonSection subtitle={'Warm Up'} />
-          <LessonSection subtitle={'Main Lesson'} />
-          <LessonSection subtitle={'Cool Down'} />
-          <LessonPlanNotes subtitle={'Notes'} />
-          <SaveButton />
-        </View>
+        <TouchableWithoutFeedback onPress={Keyboard.dismiss} backgroundColor={"orange"}>
```

Comment:
so this TouchableWithoutFeedback helps you blur the focus on the text inputs just fine, but the size of it is a little smaller than I'd like (it's not wide enough to cover the margins on the left and right of the lesson plan editor). can you look into styling for this component to see if you can make it bigger?
you can monitor its size by using the Inspector tool on the respective simulators. (also pls remove my debugging backgroundColor here uwu thanks i forgot to)